### PR TITLE
update doc

### PR DESCRIPTION
**Pull request by asm582**
doc updates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,32 @@
-## Instascale controller
+# InstaScale
 
-# High Level Steps
+InstaScale is a controller that works with Multi-cluster-app-dispatcher (MCAD) to get aggregated resources available in the kubernetes cluster without creating pending pods. It uses [machineset](https://github.com/openshift/machine-api-operator) to launch instances on cloudprovider to be added to the Kubernetes cluster.
+
+Key features:
+- Acquires aggregated heterogenous instances needed for workload execution.
+- Does not clog Kubernetes controlplane.
+- Works with your Kubernetes scheduling system to scheduler pods on aggregated resources.
```
Comment (Maxusmusti) [inline README.md:8]: *"to schedule"

File: README.md [hunk 2]
```diff
@@ -1,18 +1,32 @@
-## Instascale controller
+# InstaScale
 
-# High Level Steps
+InstaScale is a controller that works with Multi-cluster-app-dispatcher (MCAD) to get aggregated resources available in the kubernetes cluster without creating pending pods. It uses [machineset](https://github.com/openshift/machine-api-operator) to launch instances on cloudprovider to be added to the Kubernetes cluster.
```
Comment (Maxusmusti) [inline README.md:3]: *"cloud provider"

Comment (Maxusmusti) [reply README.md:3]: *machinesets

**Review by Maxusmusti — CHANGES_REQUESTED** [review]

**Comment by Maxusmusti** [comment]
Also looks like it needs a rebase (to add changes from your last PR)

**Comment by asm582** [comment]
> Also looks like it needs a rebase (to add changes from your last PR)

@Maxusmusti Thanks, I think I have addressed your comments, please review

**Review by Maxusmusti — APPROVED** [review]

**Pull request merged**